source/partials/release_notes/_release-17-4-0.md.erb

```diff
@@ -0,0 +1,108 @@
+<h4>UI Enhancements </h4>
+
+* <%= link_to_commit 'd6d59fb4a589acb7ef24416d8bb2eadea324bc3d', 'Foldable sections of the console logs.' %>
+  <figure>
+      <%= image_tag 'releases/17.4.0/expand_logs.gif', alt: 'Foldable Console Logs' %>
+  </figure>
+
+  <figure>
+      <%= image_tag 'releases/17.4.0/foldable-console-logs-1.png', alt: 'Foldable Console Logs' %>
+  </figure>
+
+  <figure>
+      <%= image_tag 'releases/17.4.0/foldable-console-logs-2.png', alt: 'Foldable Console Logs' %>
```

Comment:
Not sure what the last 2 images here do. The first one clarifies intent fairly well. Can we improve it to capture an expand on an actual task, instead of the go specific console lines?